### ISSUE TITLE
Missing check for ClassVar mismatch when a non-protocol derives from a protocol

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7455]
The following should generate an error, but it doesn't currently.

```python
from typing import ClassVar, Protocol

class Proto(Protocol):
    x: int

class Impl(Proto):
    x: ClassVar = 3
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.354, which I just published.
